### PR TITLE
Allow to transfer names

### LINKED ISSUE #1057: Assign name to a different account

### PR DESCRIPTION
**Pull request by davidyuk**
closes #1057
blocked by https://github.com/aeternity/aepp-middleware/issues/140

### CODE REVIEW

File: src/pages/mobile/NameTransfer.vue [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    }
+  },
+  methods: {
+    transferToAccount(address) {
+      this.accountTo = address;
+      this.transfer();
+    },
+    async transfer() {
+      if (!await this.$validator.validateAll()) return;
+      this.busy = true;
```
Comment (CedrikNikita) [inline src/pages/mobile/NameTransfer.vue]: I think before that line also should be dispatched confirm modal, if a user is trying to transfer to his own account.

File: src/pages/mobile/TransactionDetails.vue [hunk 2]
```diff
@@ -51,6 +51,12 @@
       :fee="transaction.tx.fee"
     />
 
+    <DetailsAddress
+      v-if="transaction.tx.accountId"
+      :name="$t('transfer.transaction.details.account')"
+      :address="transaction.tx.accountId"
+    />
+
```
Comment (CedrikNikita) [inline src/pages/mobile/TransactionDetails.vue]: Transaction details header not very informative. My proposal is to have at least transaction type in header in such cases.
image

File: src/pages/mobile/NameTransfer.vue [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+      try {
+        await this.$store.state.sdk.aensTransfer(this.nameRecord.nameHash, this.accountTo);
+        this.$store.dispatch('modals/open', {
+          name: 'notification',
+          text: this.$t('name.transfer.notification.succseed', { name: this.name }),
```
Comment (etharner) [inline src/pages/mobile/NameTransfer.vue]: Typo: succeed instead of succseed

File: src/locales/ru.json [hunk 4]
```diff
@@ -432,9 +433,20 @@
       "owner": "Владелец",
       "created-at-height": "Зарегистрировано в блоке с номером",
       "expires-at-height": "Зарегистрировано до блока с номером",
+      "to-transfer": "Передать имя",
       "to-transactions": "Открыть транзакцию"
     },
 
+    "transfer": {
+      "guide": "Передать имя которым владеет <account />",
```
Comment (etharner) [inline src/locales/ru.json]: Typo: you need to add comma before которым

**Pull request merged**